core/src/client/client.rs

```diff
@@ -286,18 +286,32 @@ impl ResealTimer for Client {
     fn set_max_timer(&self) {
         if let Some(reseal_timer) = self.reseal_timer.read().as_ref() {
             reseal_timer.cancel(RESEAL_MAX_TIMER_TOKEN).expect("Reseal max timer clear succeeds");
-            reseal_timer
+            match reseal_timer
                 .schedule_once(self.importer.miner.get_options().reseal_max_period, RESEAL_MAX_TIMER_TOKEN)
-                .expect("Reseal max timer set succeeds");
+            {
+                Ok(_) => {}
+                Err(TimerScheduleError::TokenAlreadyScheduled) => {
+                    // Since set_max_timer could be called in multi thread, ignore the TokenAlreadyScheduled error
+                }
+                Err(_) => unreachable!("Reseal max timer set succeeds"),
```

Comment:
We can't paraphrase the error message as "Expect ... succeeds, but failed". Should we change it as a negative statement?

Reply:
How about this?
`unreachable!("Reseal max timer should not fail but failed with {:?}", err)`

Reply:
Looks Good